sdk/canton/BUILD.bazel

```diff
@@ -997,7 +998,6 @@ scala_library(
         "@maven//:org_apache_pekko_pekko_actor_typed_2_13",
         "@maven//:org_postgresql_postgresql",
         "@maven//:org_reactivestreams_reactive_streams",
-        "@maven//:org_scala_lang_modules_scala_java8_compat_2_13",
```

Comment:
Don't we need this dependency? It seemed to fail without it: [https://dev.azure.com/digitalasset/daml/_build/results?buildId=180451&view=logs&j=2[…]b1-f9c1bbd1b976&t=479a56da-7cc0-515b-98b1-8e961935fd80&l=10030](https://dev.azure.com/digitalasset/daml/_build/results?buildId=180451&view=logs&j=2e1b8928-2d4f-5c49-dab1-f9c1bbd1b976&t=479a56da-7cc0-515b-98b1-8e961935fd80&l=10030)